system/application/controllers/api.php

```diff
@@ -538,6 +557,51 @@ private function _array_remap_content_update(){
 
 		return $save;
 	}
+	
+	/**
+	* _get_IIIF_metadata takes IIIF manifest url, and returns thumbnail url if one is defined
+	* @return array
+	*/
+	private function _get_IIIF_metadata($url=''){
+		$dc_check_fields = [
```

Comment:
I think the list of dcterms can be pulled from `system/application/config/rdf.php`. Is this the same list that's declared there? If it is, I think all you have to do is something along these lines:

```
$ontologies = $this->config->item('ontologies');
$dc_check_fields = $ontologies['dcterms'];
```

Reply:
Good find Artie - yeah, probably best to just rely on the canonical list there.

Reply:
Ah, thanks for the heads up!